### ISSUE TITLE
 [Bug] I don't see a player in the tab because of the esp 

### ISSUE THREAD

**Issue by F3F5** [Photon-GitHub/AntiCheatAddition#642]
![331897910-238e4748-a061-4ba7-847e-eb478f8e81da](https://github.com/Photon-GitHub/AntiCheatAddition/assets/170719238/5153446f-15df-435a-ada5-dd756984a84c)

[21:57:09 INFO]: [AntiCheatAddition] Version: 9.1.4

[21:57:14 INFO]: This server is running PandaSpigot version git-PandaSpigot-119 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT) (Git: 858ab0f)
[21:57:14 INFO]: Checking version, please wait...
[21:57:15 INFO]: You are running the latest version

**Comment by Photon-GitHub** [comment]
> @Photon-GitHub if (skin == null || skin.isEmpty()) return; cause the problem on 1.8 servers. Maybe it should be removed?

No, because this is needed on newer versions, otherwise errors will appear. A potential solution would be an empty list, but I would need to test it first.
